tracer/build/_build/docker/reliability-env/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS dotnet_agent_debug
+ARG LINUX_PACKAGE
+
+RUN apt-get update && apt-get install dos2unix
+
+COPY ${LINUX_PACKAGE} /binaries/
+RUN ls /binaries/datadog-dotnet-apm-*.tar.gz | cut -d - -f 4 | sed -e 's/\.tar.gz$//' > /binaries/LIBRARY_VERSION
+RUN mv /binaries/${LINUX_PACKAGE} /binaries/datadog-dotnet-apm.tar.gz
+
+COPY ddtrace-info.sh query-versions.fsx /binaries/
+RUN dos2unix /binaries/ddtrace-info.sh
+RUN /binaries/ddtrace-info.sh
+
+RUN touch -c -a -m -d @0 /binaries/*
+
+FROM scratch as collect
+COPY --from=dotnet_agent_debug /binaries/datadog-dotnet-apm.tar.gz /datadog-dotnet-apm.tar.gz
+COPY --from=dotnet_agent_debug /binaries/LIBRARY_VERSION /LIBRARY_VERSION
+COPY --from=dotnet_agent_debug /binaries/LIBDDWAF_VERSION /LIBDDWAF_VERSION
+COPY --from=dotnet_agent_debug /binaries/APPSEC_EVENT_RULES_VERSION /APPSEC_EVENT_RULES_VERSION
```

Comment:
This all seems to be somewhat convoluted ways of extracting the library version. I think that's because it's copied from the system tests. 

But as this file is in this repo, we _know_ all those values statically, so we can just embed the values here directly I think (e.g. we have a task that automatically bumps versions in the library using a simple regex). That would simplify all this, remove the need for the dotnet base image etc etc.